content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,12 @@
+## Bug fixes and minor enhancements
```

Comment:
How about we completely separate bug fixes from enhancements? Then you don't need to say "Fix:" everywhere.